plaid/sandbox.go

```diff
@@ -64,3 +76,34 @@ func (c *Client) ResetSandboxItem(accessToken string) (resp ResetSandboxItemResp
 	err = c.Call("/sandbox/item/reset_login", jsonBody, &resp)
 	return resp, err
 }
+
+
+func (c *Client) SetSandboxItemVerificationStatus(accessToken string, accountID string, verificationStatus string) (resp SetSandboxItemVerificationStatusResponse, err error) {
```

Comment:
nitty: I don't think you need to name the return vars unless you do a naked return

Reply:
...though this matches the convention in this file, so take it or leave it.